parts/windows/csecmd.ps1

```diff
@@ -1,22 +1,19 @@
-# This csecmd is not used until servicePrincipalClientSecret is encoded,
-# keeping this file for now to further facilicate the future removing ARM Template
 echo %DATE%,%TIME%,%COMPUTERNAME% && powershell.exe -ExecutionPolicy Unrestricted -command \"
 $arguments = '
--MasterIP {{ GetKubernetesEndpoint }} 
```

Comment:
As this command will finally be broken into one line, and the newline will be replaced with blank, so we remove the redundant blank in this file, otherwise, we'll have two consecutive blanks